src/Switch-Arduino_MCP4728/main.py

```diff
@@ -158,6 +158,17 @@ def configure(self):
             self.set_vref(use_internal_vref=False)
             self.max_voltage = self.external_voltage
 
+    def unconfigure(self):
+        # Set all outputs to 0
+        if self.multi_pins:
+            for address in self.addresses:
+                self.set_address(address)
+                for pin in range(4):
+                    print(pin)
```

Comment:
This print could be removed to keep the Debug widget empty if there are no problems.